### ISSUE TITLE
Postfix: Enable SSL if ssl/certs contains any files

### ISSUE THREAD

**Issue by ralflang** [maintaina/containers#2]
```
smtpd_use_tls = no
smtpd_tls_security_level = none
smtpd_tls_ask_ccert = no
smtpd_tls_received_header = no
smtpd_tls_chain_files =
```

**Comment by ikselven** [comment]
So the entrypoint script should check for certs in a predefined location inside the container and if it finds certs configure TLS for Postfix else disable TLS?

**Comment by ralflang** [comment]
Yes. The idea is to provide a mountpoint for a cert or possibly set of certs and have the container autoconfigure based on that, with very limited need to also provide a custom config for most use cases.
